tests/integration/annotation_import/test_bulk_import_request.py

```diff
@@ -370,3 +370,69 @@ def test_nested_video_object_annotations(client,
     import_annotations.wait_until_done()
 
     assert import_annotations.errors == []
+
+
+def test_below_annotation_limit_on_single_data_row(
+        client, configured_project_with_one_data_row, video_data, rand_gen):
+    _, data_row_uids = video_data
+
+    def create_label():
+        return Label(data=VideoData(uid=data_row_uids[0]),
+                     annotations=[
+                         VideoObjectAnnotation(name="bbox",
+                                               keyframe=True,
+                                               frame=4,
+                                               segment_index=0,
+                                               value=Rectangle(
+                                                   start=Point(x=100, y=100),
+                                                   end=Point(x=105, y=105),
+                                               ))
+                     ])
+
+    configured_project_with_one_data_row.update(media_type=MediaType.Video)
+    configured_project_with_one_data_row.create_batch(
+        rand_gen(str),
+        data_row_uids,  # sample of data row objects
+        5  # priority between 1(Highest) - 5(lowest)
+    )
+    labels = [create_label() for _ in range(5000)]
```

Comment:
Are you considering running a full upload of 5,000 annotations for SDK integration testing? This could significantly impact performance and might even result in flakiness. I recall we previously reduced some tests due to similar concerns. Would you consider creating a method or perhaps patching the LABEL_LIMIT to a smaller number, like 2, specifically for integration tests to mitigate these issues?